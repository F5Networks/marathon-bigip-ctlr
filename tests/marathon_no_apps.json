[
    {
        "acceptedResourceRoles": null,
        "args": [
            "--sse",
            "--marathon",
            "http://10.141.141.10:8080",
            "--partition",
            "mesos",
            "--hostname",
            "10.128.1.145",
            "--username",
            "admin",
            "--password",
            "default"
        ],
        "backoffFactor": 1.15,
        "backoffSeconds": 1,
        "cmd": null,
        "constraints": [],
        "container": {
            "docker": {
                "forcePullImage": false,
                "image": "f5networks/marathon-bigip-ctlr:latest",
                "network": "BRIDGE",
                "parameters": [],
                "privileged": false
            },
            "type": "DOCKER",
            "volumes": []
        },
        "cpus": 0.5,
        "dependencies": [],
        "deployments": [],
        "disk": 0,
        "env": {},
        "executor": "",
        "healthChecks": [],
        "id": "/f5-service-router",
        "instances": 1,
        "labels": {},
        "maxLaunchDelaySeconds": 3600,
        "mem": 128,
        "ports": [
            10000
        ],
        "requirePorts": false,
        "storeUrls": [],
        "tasks": [
            {
                "appId": "/f5-service-router",
                "host": "10.141.141.10",
                "id": "f5-service-router.9c824d13-3890-11e6-a960-0242ed8b262a",
                "ports": [
                    31324
                ],
                "slaveId": "c0ca17c6-075f-4e12-997a-38537feb6c16-S0",
                "stagedAt": "2016-06-22T15:47:23.399Z",
                "startedAt": "2016-06-22T15:47:43.210Z",
                "version": "2016-06-22T15:47:22.754Z"
            }
        ],
        "tasksHealthy": 0,
        "tasksRunning": 1,
        "tasksStaged": 0,
        "tasksUnhealthy": 0,
        "upgradeStrategy": {
            "maximumOverCapacity": 1,
            "minimumHealthCapacity": 1
        },
        "uris": [],
        "user": null,
        "version": "2016-06-22T15:47:22.754Z",
        "versionInfo": {
            "lastConfigChangeAt": "2016-06-22T15:47:22.754Z",
            "lastScalingAt": "2016-06-22T15:47:22.754Z"
        }
    }
]
