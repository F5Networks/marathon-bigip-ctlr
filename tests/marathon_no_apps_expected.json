{"pools": [], "iapps": [], "l7Policies": [], "virtualServers": [], "monitors": []}
